app/components/autocomplete/index.js

```diff
@@ -4,16 +4,20 @@
 import {connect} from 'react-redux';
 
 import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
+import {isMinimumServerVersion} from 'mattermost-redux/utils/helpers';
 
 import {getDimensions} from 'app/selectors/device';
 
 import Autocomplete from './autocomplete';
 
 function mapStateToProps(state) {
     const {deviceHeight} = getDimensions(state);
+    const serverVersion = state.entities.general.serverVersion;
+    const enableDateSuggestion = isMinimumServerVersion(serverVersion, 5, 2);
```

Comment:
I don't think 5.2 is the right version, 5.2 is the one currently out so the next one would be 5.3 and only if the server side changes makes it to the release, but if don't then we could update this to be 5.4 later

Reply:
I set it to 5.2 for now, there's no way to test the feature otherwise, I would update the version when ready to release